articles/event-grid/includes/schema-app-configuration.md

```diff
@@ -54,12 +56,22 @@ An event has the following top-level data:
 
 The data object has the following properties:
 
+# [Key-value event](#tab/key-value-event-data-object)
+
 | Property | Type | Description |
 | -------- | ---- | ----------- |
 | `key` | string | The key of the key-value that was modified or deleted. |
 | `label` | string | The label, if any, of the key-value that was modified or deleted. |
 | `etag` | string | For `KeyValueModified` the etag of the new key-value. For `KeyValueDeleted` the etag of the key-value that was deleted. |
```

Comment:
we have sync token on kv event. Looks like it was never updated.

Reply:
yes, there are 4 fields in data object of kv event, and etag was missed here, I will update it as well